src/redis_request.cc

```diff
@@ -36,13 +37,17 @@ Status Request::Tokenize(evbuffer *input) {
             multi_bulk_len_ = std::stoull(std::string(line + 1, len-1));
           } catch (std::exception &e) {
             free(line);
-            return Status(Status::NotOK, "Protocol error: expect integer");
+            return Status(Status::NotOK, "Protocol error: invalid multibulk length");
+          }
+          if (multi_bulk_len_ > PROTO_MULTI_MAX_SIZE) {
+            free(line);
+            return Status(Status::NotOK, "Protocol error: invalid multibulk length");
           }
           state_ = BulkLen;
         } else {
           if (len > PROTO_INLINE_MAX_SIZE) {
             free(line);
-            return Status(Status::NotOK, "Protocol error: too big inline request");
+            return Status(Status::NotOK, "Protocol error: invalid bulk length");
```

Comment:
so is this change intentional? i see the arg name saying INLINE, just trying to confirm